app/avo/concerns/auditable.rb

```diff
@@ -39,7 +39,7 @@ def in_audited_transaction(auditable:, admin_github_user:, action:, fields:, arg
           changes = merge_changes!(changes, record.attributes.slice("id").transform_values { [_1, _1] }) if changes.key?("id")
           changes = merge_changes!(changes, record.attributes.compact.transform_values { [_1, nil] }) if record.destroyed?
 
-          [key, { changes:, unchanged: record.attributes.except(*changes.keys) }]
```

Comment:
Rubocop side-effect. Not related to this change but necessary to keep lint passing (while also improving how expect syntax is used)